jobs/release-microk8s-capi.yaml

```
@@ -0,0 +1,60 @@
+# Builds and releases microk8s capi providers
+
+- job:
+    name: 'release-microk8s-capi'
+    description: |
+      Build and release MicroK8s capi.
+    project-type: freestyle
+    node: runner-amd64
```

Use `scm` to automatically include the jenkins repo as the `$WORKSPACE`
```suggestion
    node: runner-amd64
    scm:
      - k8s-jenkins-jenkaas
```

this lets you remove the `git clone https://github.com/charmed-kubernetes/jenkins.git -b main jenkins` below